x-pack/metricbeat/module/sql/query/query_integration_test.go

```diff
@@ -27,46 +27,62 @@ import (
 	"github.com/elastic/beats/v7/metricbeat/module/postgresql"
 )
 
-type testFetchConfig struct {
+type config struct {
```

Comment:
As this `config` type is only relevant to tests, I'd maintain the old naming so you won't look at it later and thing it's something in the main code. But that's a subjective opinion.